Examples/DataRepresentation/Image/Image4.cxx

```diff
@@ -347,8 +346,8 @@ int main(int, char *[])
 
   // Software Guide : BeginCodeSnippet
   const ImageType::IndexType LeftEyeIndex = GetIndexFromMouseClick();
-  ImageType::PointType LeftEyePoint;
-  image->TransformIndexToPhysicalPoint(LeftEyeIndex,LeftEyePoint);
+  ImageType::PointType       LeftEyePoint;
```

Comment:
Since clang-format doesn't distinguish between in-class and in-function declarations, maybe we should abandon alignment of declarations? Alignment will increase diff sizes for simple and style changes - not problematic, purely annoying.

Reply:
Abandoned in new code.

Reply:
As I was initially reviewing, you were making changes. Parallel synchronization of humans is non-trivial 🤖